judge/views/contests.py

```diff
@@ -97,13 +100,16 @@ def get_context_data(self, **kwargs):
                     active.append(participation)
                     present.remove(participation.contest)
 
-        active.sort(key=attrgetter('end_time'))
+        active.sort(key=attrgetter('end_time', 'start_time'))
```

Comment:
Sorry, I wasn't clear with what I meant. I meant something similar with https://github.com/DMOJ/online-judge/pull/1474#discussion_r449338840 and having `key` as the secondary order.

Reply:
Like this?

Reply:
Yup!